recipes/GeneralisedFormanRicci/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "GeneralisedFormanRicci" %}
+{% set version = "0.0.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/GeneralisedFormanRicci-{{ version }}.tar.gz
+  sha256: 803d78e2fbad78ee5e0353978cbfbfe6023eb6f4f08f56183f7614b10286fbc6
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - scipy
+    - networkx
+    - gudhi
+    - python >=3.6
+  run:
+    - scipy
+    - networkx
+    - gudhi
+    - python >=3.6
+
+test:
+  imports:
+    - GeneralisedFormanRicci
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ExpectozJJ/GeneralisedFormanRicci
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE
```

Comment:
Add comment:

```yaml
# License file manually packaged. See https://github.com/ExpectozJJ/GeneralisedFormanRicci/pull/2
license_file: LICENSE
```

This will be a reminder to check the status of the license file in the source distribution in the future. If my PR upstream (https://github.com/ExpectozJJ/GeneralisedFormanRicci/pull/2) gets merged or a different solution is found, then once that release is updated in the feedstock, the manually packaged license can be removed as well as the comment.